<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportConsensusDialog</class>
 <widget class="QDialog" name="ExportConsensusDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>221</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export Consensus</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="filepathLabel">
       <property name="text">
        <string>Export to file</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="filepathLineEdit"/>
     </item>
     <item row="0" column="3">
      <widget class="QToolButton" name="filepathToolButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2" colspan="2">
      <widget class="QComboBox" name="documentFormatComboBox"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="sequenceNameLabel">
       <property name="text">
        <string>Sequence name</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2" colspan="2">
      <widget class="QLineEdit" name="sequenceNameLineEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="algorithmLabel">
       <property name="text">
        <string>Consensus algorithm</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2" colspan="2">
      <widget class="QComboBox" name="algorithmComboBox"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="documentFormatLabel">
       <property name="text">
        <string>File format</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="variationModeLabel">
       <property name="text">
        <string>Mode</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2" colspan="2">
      <widget class="QComboBox" name="variationModeComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="keepGapsCheckBox">
     <property name="text">
      <string>Keep gaps</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="addToProjectCheckBox">
       <property name="text">
        <string>Add to project</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
